relay-metrics/src/protocol.rs

```diff
@@ -164,20 +165,92 @@ fn is_valid_name(name: &str) -> bool {
     false
 }
 
-/// Validates an MRI of the form `<ty>:<ns>/<name>@<unit>`
+/// The namespace of a metric.
 ///
-/// Note that the format used in the statsd protocol is different: Metric names are not prefixed
-/// with `<ty>:` as the type is somewhere else in the protocol.
-pub(crate) fn is_valid_mri(name: &str) -> bool {
-    let mut components = name.splitn(2, ':');
-    if components.next().is_none() {
-        return false;
+/// This successfully deserializes any kind of string, but in reality only `"sessions"` and
+/// `"transactions"` is supported. Everything else is dropped both in the metrics aggregator and in
+/// the store actor.
```

Comment:
Please also explain what the namespace _is_. Especially which namespaces are supported is implicit by looking at the variants of the enum.